### PR TITLE
Two Factor Auth 

### PR DESCRIPTION
**Pull request by ashanaakh**

### CODE REVIEW

**Comment by ashanaakh** [comment]
Ready, except styles for page. Waiting for @gfedorenko fix that.

**Comment by mod** [comment]
Try to encrypt the field otp_seed
With bcrypt

**Comment by ashanaakh** [comment]
@mod Working on vault

**Comment by ashanaakh** [comment]
@mod you can merge this.

**Comment by gfedorenko** [comment]
Can you rebase and squash?

File: app/controllers/account_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+#
+# AccountController
+#
```
Comment (mod) [inline app/controllers/account_controller.rb]: what is it for ?

File: app/models/account.rb [hunk 2]
```diff
@@ -15,6 +17,23 @@ class Account < ApplicationRecord
 
   validates :email, uniqueness: true
 
+  # Generates new url
```
Comment (mod) [inline app/models/account.rb]: what is account.url ?

**Comment by ashanaakh** [comment]
@mod fixed

**Pull request merged**